e2e/basic/common.go

```diff
@@ -66,8 +65,6 @@ func (bt *FlowCaptureTester) DoTest(t *testing.T) {
 			assert.Less(t, flow["Packets"], float64(10))
 			assert.Less(t, time.Since(asTime(flow["TimeFlowEndMs"])), 15*time.Second)
 			assert.Less(t, time.Since(asTime(flow["TimeFlowStartMs"])), 15*time.Second)
-
-			assert.NotEmpty(t, flow["Interface"])
```

Comment:
I don't know why yet but I'm getting <nil> in `IfDirections` and `Interfaces` here :thinking: 

I'm investigating